templates/vue/cypress/integration/settings.spec.js

```diff
@@ -0,0 +1,68 @@
+import { TEST_TAPESTRY_NAME } from "../support/constants"
+
+describe("Settings", () => {
+  beforeEach(() => {
+    cy.fixture("stump.json").as("tapestry")
+    cy.setup("@tapestry")
+    cy.getByTestId("settings-button").click()
+  })
+
+  it(`should be able to set default depth`, () => {
```

Comment:
Yes, I mentioned this in the description:

> I was using the "Given-When-Then" methodology to name tests, but I quickly found that it became tedious to create clear test names - especially since in E2E tests there are a lot of different assertions and events happening in one run. In one of the spec files, `settings.spec.js`, I used the normal convention but I didn't change the naming for the other tests that were already written.

What are your thoughts?

Reply:
Oh sorry I didn't see that. I think it would make sense to use the same naming convention, so I suggest either changing the old names or keep following Given-When-Then. That said, if you think it takes too long and it doesn't matter for some reason, I trust your judgement.